index.js

```diff
@@ -41,6 +42,19 @@ function ldnode (argv) {
     // Setting options as local variable
     app.locals.ldp = opts;
 
+    // Setting CORS
+    app.use(cors({
+        methods: [
+            'OPTIONS', 'HEAD', 'GET',
+            'PATCH', 'POST', 'PUT', 'MKCOL', 'DELETE',
+            'COPY', 'MOVE', 'LOCK', 'UNLOCK'
```

Comment:
Actually cors() implements a trivial 'OPTIONS' handler by default - but we have to linkeddata-ize it!